dapps/digital-twin-detail/src/components/sharing/SharingOverview.ts

```diff
@@ -0,0 +1,234 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+// vue imports
+import Component, { mixins } from 'vue-class-component';
+
+// evan.network imports
+import { EvanComponent } from '@evan.network/ui-vue-core';
+import { DAppTwin, DAppContainer } from '@evan.network/digital-twin-lib';
+import { profileUtils } from '@evan.network/ui';
+
+import ShareContainerComponent from '../container/ShareContainer';
+
+/**
+ * Used to start user name / type loading directly while container resolve.
+ */
+interface LoadingSharedUserInterface {
+  containers: Array<string>;
+  name: Promise<string>;
+  type: Promise<string>;
+}
+
+/**
+ * Format of resolved "search" results
+ */
+interface SharedUserInterface {
+  address: string; // account address
+  containerNames: Array<string>; // list of containers, mapped to it's names
+  containers: Array<string>; // list of containers, for that the user is permitted;
+  icon: string; // account type material icon
+  name: string; // users name, alias or account address
+  type: string; // account type
+}
+
+interface ColumnInterface {
+  key: string;
+  label: string;
+  sortable?: boolean;
+  tdClass?: string;
+  thClass?: string;
+}
+
+@Component({ })
+export default class SharingOverview extends mixins(EvanComponent) {
+  columns: ColumnInterface[ ] = [];
+
+  data: SharedUserInterface[ ] = [];
+
+  isActiveSearch = false;
+
+  loading = true;
+
+  reverse = false;
+
+  selectedFilter = 'all';
+
+  searchTerm = '';
+
+  sortBy = 'name';
+
+  filters = [
+    { type: 'unshared', icon: 'help-circle-outline' },
```

Comment:
this is additionally to the figma UI right? Is the icon communicated with UI Team?